createjson.rb

```diff
@@ -0,0 +1,58 @@
+require 'json'
+
+class CreateJsonBook
```

Comment:
Great job team🤝

- [ ] As stated in the project requirements, kindly ensure on exit from the student's app make sure the data is being saved (all the data: books, people, and rentals) into JSON files. However, in your case, these files are not created rather the app breaks, and no data is saved. Kindly make sure this issue is fixed.
![school](https://github.com/Kaghenimbale/school_library/assets/63917725/2c9d0df3-9867-4398-a5c8-0a6780d2527d)

Reply:
well resolved! ✔️